stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -89,9 +88,7 @@ spec:
         {{- if .Values.livenessProbe.enabled }}
         livenessProbe:
           exec:
-            command:
-            - rabbitmqctl
-            - status
+            command: ["sh", "-c", "curl -f --user {{ .Values.rabbitmq.username }}:$RABBITMQ_PASSWORD 127.0.0.1:{{ .Values.rabbitmq.managerPort }}/api/healthchecks/node"]
```

Comment:
this endpoints returns 200 even if it detects an error, see https://github.com/helm/charts/issues/7752